src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -2080,17 +2083,17 @@ static void top_toolbar_tool_update_land(int16_t x, int16_t y)
     }
 
     int16_t tool_size = gLandToolSize;
-    LocationXY16 mapTile;
+    CoordsXY mapTile;
     uint8_t side;
 
     gMapSelectFlags &= ~MAP_SELECT_FLAG_ENABLE;
     if (tool_size == 1)
     {
         int32_t selectionType;
         // Get selection type and map coordinates from mouse x,y position
-        CoordsXY mapCoords = screen_pos_to_map_pos(x, y, &selectionType);
+        CoordsXY mapCoords = screen_pos_to_map_pos({ x, y }, &selectionType);
```

Comment:
mapCoords is never actually used. This function doesn't need to save the result it is only been used to get selectionType.